src/internal_events/mod.rs

```diff
@@ -126,6 +128,8 @@ pub use self::syslog::*;
 #[cfg(feature = "transforms-tag_cardinality_limit")]
 pub(crate) use self::tag_cardinality_limit::*;
 pub use self::tcp::*;
+#[cfg(feature = "transforms-tokenizer")]
+pub use self::tokenizer::*;
```

Comment:
Using `pub(crate)` here and in `tokenizer.rs` will help with inlining

Reply:
Adding here, but do we need to propagate it down? I assumed not, but now I'm less sure.

Separately, I was also thinking of trying to make the `internal_events` module itself `pub(crate)`, which seems like a good thing to do if propagation works the way I assumed.

Reply:
Yes, it would need to be on both the actual structs and the re-import, as without the structs being restricted, they are still fully `pub`.

Making the module itself `pub(crate)` is an good idea try, and it should achieve the same effect as restricting everything inside. I tested with the playground, and it seems to achieve the desired effect.

Reply:
Ah, right, there is still a fully `pub` path to the item. Will fix.